aura-cloud-dbaas/aura-cloud-dbaas.adoc

```diff
@@ -51,6 +51,11 @@ link:/developer/aura-data-import[Importing data into Neo4j Aura, role="button fe
 
 // * Import from Web APIs
 
+[deploying-applications]
+== Deploying applications to Aura
+
+* link:/developer/aura-grandstack[Deploying a GRANDstack application^]
+
```

Comment:
We also need to add this page to the menu-partial, right? That way, it will appear in the left-side menu. :)

Reply:
Good point, I forgot to add it